app/solution/page.tsx

```
@@ -118,30 +118,35 @@ const solutionSections: {
+            <p className="text-sm sm:text-base md:text-lg lg:text-xl xl:text-2xl text-gray-600 dark:text-gray-400 max-w-xs sm:max-w-md md:max-w-2xl lg:max-w-4xl xl:max-w-5xl mx-auto leading-relaxed px-3 sm:px-4 md:px-0">
+              Tradoxus solves the critical problems in crypto trading education
+              through a gamified educational web-app that provides safe,
+              practical learning experiences.
+            </p>
+          </div>
+
+          {/* Features Grid - Responsive across all screen sizes */}
+          <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-3 2xl:grid-cols-4 gap-3 sm:gap-4 md:gap-6 lg:gap-8 xl:gap-10">
+            {solutionSections.map((section, index) => (
+              <SectionCard
+                key={index}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)